### ISSUE TITLE
[HIGH] Missing ARIA Labels - All 28 prediction components need a11y

### ISSUE THREAD

**Issue by xkonjin** [xkonjin/xUSDT#196]
See PRODUCTION_READINESS_PLAN.md for details
